packages/@ember/-internals/runtime/lib/mixins/array.js

```diff
@@ -1641,12 +1641,36 @@ let A;
 
 if (ENV.EXTEND_PROTOTYPES.Array) {
   NativeArray.apply(Array.prototype);
-  A = arr => arr || [];
+
+  A = function(arr) {
+    deprecate(
+      '`new A()` has been deprecated, please update to calling A as a function: `A()`',
+      !(this instanceof A),
+      {
+        id: 'array.new-array-wrapper',
+        until: '3.9.0',
+        url: 'https://emberjs.com/deprecations/v3.x#toc_new-array-wrapper',
```

Comment:
Seems to be wrong (the url doesn't link directly), when I find the right link it looks like:

https://deprecations-app-prod.herokuapp.com/deprecations/v3.x/#toc_array-new-array-wrapper

Can you confirm which is correct?

Reply:
you are correct! Updated